chains/evm/cli/erc20/deposit.go

```diff
@@ -33,7 +33,10 @@ func BindDepositCmdFlags(cli *cobra.Command) {
 	cli.Flags().String("destId", "", "destination domain ID")
 	cli.Flags().String("resourceId", "", "resource ID for transfer")
 	cli.Flags().Uint64("decimals", 0, "ERC20 token decimals")
-	cli.MarkFlagRequired("decimals")
+	err := cli.MarkFlagRequired("decimals")
+	if err != nil {
```

Comment:
Why are we handling this `error` here, but changes in `chains/evm/cli/admin/set-deposit-nonce.go` just ignore this error.

I think both ways are fine, as this error can only happen if the flag doesn't exist - but we just need to choose how are we handling this and be consistent.

Reply:
done